pipelines/offlineinstaller/Jenkinsfile

```diff
@@ -8,76 +7,58 @@ node() {
-                	values.put('jobName', jobName)
-                    	values.put('ansiblePlaybook', ansiblePlaybook)
-                    	values.put('ansibleExtraArgs', ansibleExtraArgs)
-                    	println values
-                    	ansible_playbook_run(values)
+                stage('get artifact') {
+                    values = lp_dp_params()
+                    currentWs = sh(returnStdout: true, script: 'pwd').trim()
+                    artifact = values.artifact_name + ":" + values.artifact_version
+                    values.put('currentWs', currentWs)
+                    values.put('artifact', artifact)
+                    artifact_download(values)
+                    sh """
+                       unzip ${artifact}
+                       tar -xvzf app_dist.tar.gz
+                    """
                 }
-		try {
-		     stage('Build Installer, create and upload it to azure') {
-			dir('offline-installer-repo') {
-			    sh """
-                               bash -x build.sh
-		             """
-			}
-			values = [:]
-                        ansiblePlaybook = "${currentWs}/ansible/offline-installer.yml --vault-password-file /var/lib/jenkins/secrets/vault-pass "
-                        ansibleExtraArgs = " --extra-vars \"offline_repo_location=$currentWs uploadInstaller=True offline_installer_type=${offline_installer_type}\""
-                        values.put('currentWs', currentWs)
-                        values.put('env', envDir)
-                        values.put('module', module)
-                        values.put('jobName', jobName)
-                        values.put('ansiblePlaybook', ansiblePlaybook)
-                        values.put('ansibleExtraArgs', ansibleExtraArgs)
-                        println values
-                        ansible_playbook_run(values)
-			archiveArtifacts artifacts: '*.zip', onlyIfSuccessful: true
-			archiveArtifacts artifacts: 'latest.json', onlyIfSuccessful: true
-		     }
-		}
-		catch (err) {
-			values = [:]
-                        ansiblePlaybook = "${currentWs}/ansible/offline-installer.yml --vault-password-file /var/lib/jenkins/secrets/vault-pass "
-                        ansibleExtraArgs = " --extra-vars \"offline_repo_location=$currentWs removeOfflineInstallerFolder=True offline_installer_type=${offline_installer_type}\""
-                        values.put('currentWs', currentWs)
-                        values.put('env', envDir)
-                        values.put('module', module)
-                        values.put('jobName', jobName)
-                        values.put('ansiblePlaybook', ansiblePlaybook)
-                        values.put('ansibleExtraArgs', ansibleExtraArgs)
-                        println values
-                        ansible_playbook_run(values)
-                        currentBuild.result = 'SUCCESS'
-		     }
-        }
+                stage('Install the offline desktop Application') {
+                      values = [:]
```

Comment:
This will overrite the map to empty which has data from here - https://github.com/project-sunbird/sunbird-devops/pull/2075/files#diff-45f59f142fc6d4fa9637a40b5540bb4f2c568336a2419e1d7b3d13aca6391ff7R26